src/skins/vector/css/common.css

```diff
@@ -62,18 +57,14 @@ input[type=text]:focus, textarea:focus {
     box-shadow: none;
 }
 
-/* Prevent ugly dotted highlight around selected elements in Firefox */
```

Comment:
`-moz-appearance: none;` in the `reset.css` should resolve any nasty moz presentation